frontend/cmd/cmd.go

```diff
@@ -79,6 +83,9 @@ func NewRootCmd() *cobra.Command {
 		},
 	}
 
+	rootCmd.Flags().StringVar(&opts.auditTCPAddr, "audit-tcp-addr", os.Getenv("AUDIT_TCP_ADDR"), "OTEL Address to send audit logging to")
```

Comment:
This is a personal preference nit, but could we spell out "address" instead of abbreviating?  Here and everywhere else it appears.  So `auditTCPAddress` and `"audit-tcp-address"` and `"AUDIT_TCP_ADDRESS"`.